junction/devices/models.py

```diff
@@ -14,7 +13,7 @@ def expiry_time(expiry_mins=60):
 
 
 class Device(TimeAuditModel):
-    uuid = UUIDField(version=1, hyphenate=True, unique=True, db_index=True)
+    uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
```

Comment:
> Okay, so we'll move to Django's UUIDField and figure out a way to migrate old data to this.

@pradyunsg would appreciate some help around this.

So I replaced the `django-uuidfield` with Django's UUIDField which created a new migration. But the old migration of this field still exists which will throw error as the `uuidfield` package doesn't exist anymore. What do you think would be the best way to handle this scenario, currently I am commenting out the lines which uses that package in the old migration.

Also, how do I migrate the new migrations, as `uuid` column for this model already exists. And while doing so how do I make sure that the old data of that field is migrated to the new one.

I looked it up online but couldn't find anything on how to tackle this problem, some guidance/hints would be great. :)

Reply:
Okay, so this is resolved, as I got to know that it's better to edit the old migration files to pretend that the field was always a models.UUIDField. That way we won't require any migration. Checkout the answer at [Django Forum](https://forum.djangoproject.com/t/migrating-a-custom-uuidfield-to-djangos-uuidfield/1883) for more clarification.